test/SimpleWebRequest.spec.ts

```diff
@@ -98,14 +99,16 @@ describe('SimpleWebRequest', () => {
         const method = 'POST';
         const url = faker.internet.url();
 
-        new SimpleWebRequest<string>(url, method, {}, () => headers).start();
-
-        const request = jasmine.Ajax.requests.mostRecent();
-
-        expect(request.requestHeaders['X-Requested-With']).toEqual(headers['X-Requested-With']);
-        expect(request.requestHeaders['Max-Forwards']).toEqual(headers['Max-Forwards']);
+        let request: any;
+        setTimeout(() => {
+            request = jasmine.Ajax.requests.mostRecent();
+            request.respondWith({ status: 200 });
+        }, 0);
 
-        request.respondWith({ status });
+        return new SimpleWebRequest<string>(method, url, {}, () => headers).start().then(() => {
+            expect(request.requestHeaders['X-Requested-With']).toEqual(headers['X-Requested-With']);
+            expect(request.requestHeaders['Max-Forwards']).toEqual(headers['Max-Forwards']);
+        });
     });
 
     it('forbids to set Accept header', () => {
```

Comment:
Not using `done` or `return` but test is (or can be) async now, right? [Both `'forbids to set Accept header'` and `forbids to set Content-Type header`.]